docs/interfaces/hmx/digital_io.md

```diff
@@ -12,23 +12,35 @@ The HMX-specific Digital I/Os are listed below. Check the technical file for mor
 There are four different types of digital outputs.
 
 - outputs with ability to sink a signal to ground
-    - SINK_OUT_1
-    - SINK_OUT_2
-- Activate eth2 network on OBDII cable
-    - Digital_output_ETH2_ACT
-- outputs with ability to source VCC power (they have different maximum current)
-    - SOURCE_Digital_OUT_1
-    - SOURCE_Digital_OUT_2
-    - SOURCE_OUT_3
-    - SOURCE_OUT_4
-    - SOURCE_OUT_5
-    - SOURCE_OUT_6
-    - SOURCE_OUT_7
-    - SOURCE_OUT_8
-    - SOURCE_PWR_OUT_LED_1
-    - SOURCE_PWR_OUT_LED_2
+- outputs with ability to source VCC power (they have different maximum current). Either controlled by software or always on.
```

Comment:
outputs with the ability to source VCC power (they have different maximum current), either controlled by software or always on.